analytical_engine/core/grape_instance.cc

```diff
@@ -908,6 +908,7 @@ bl::result<std::shared_ptr<DispatchResult>> GrapeInstance::OnReceive(
   switch (cmd.type) {
   case rpc::CREATE_GRAPH: {
     BOOST_LEAF_AUTO(graph_def, loadGraph(params));
+    syncGraphDef(graph_def);
```

Comment:
This is the ad-hoc way to sync the result for graph only. I think we have discussed doing this by setting the `AggregatePolicy` of the result, and do this in the aggregate function, which is defined in `graphscope_service.cc`. And the new policy can be named `kMerge`